### PR TITLE
This adds 32bit configuration for chromium web browser- (Solves -#426)

### LINKED ISSUE #426: Adding 32-bit differentiation for chromium web browser
- Issue type: Feature request 

### Short description

Currently, only chromium for 64bit processors are installed in the meilix custom ISO, opposed to the fact that we are providing option for using 32bit differentiation in generator webapp.
This option needs to be added for the meilix 32bit version.

### Usecase

Provide an optio  to install chromium 32bit version in the meilix ISO, depending upon the choice of user to use 32bit architecture or 64bit architecture.

### PR DESCRIPTION
**Pull request by pranav1698**
### Short description
This adds the 32bit configuration for the chromium web browser

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

Fixes #426

### CODE REVIEW

File: scripts/packages/chromium-package.sh [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
+# Installing chromium 32 bit or 64 bit browser 
+if [ "$arch" == "i386" ];
+then 
+wget -q security.ubuntu.com/ubuntu/pool/universe/c/chromium-browser/chromium-browser_71.0.3578.98-0ubuntu0.16.04.1_i386.deb -O chromium-browser_i386.deb
```
Comment (abishekvashok) [inline scripts/packages/chromium-package.sh]: Won't `sudo apt-get -qq -y install chromium-browser:i386` work?
I am seeing an hard coded version number on the url and I guess this won't install the latest 32 bit package **if** a new update comes out

Comment (pranav1698) [reply scripts/packages/chromium-package.sh]: @abishekvashok I will test one of the releases with the following command and get back
I was also thinking of another alternative that could work

File: sources.xenial.list [hunk 2]
```diff
@@ -10,5 +10,8 @@ deb-src http://archive.ubuntu.com/ubuntu xenial-security main restricted univers
 deb http://archive.ubuntu.com/ubuntu xenial-backports main restricted universe multiverse
 deb-src http://archive.ubuntu.com/ubuntu xenial-backports main restricted universe multiverse
 
+#This is for 32 bit version updates
+deb http://security.ubuntu.com/ubuntu xenial-security main universe
```
Comment (abishekvashok) [inline sources.xenial.list]: I doubt we need this additional source repository, this repo contains security fixes done by Ubuntu maintainers for packages that are found in the `archive.ubuntu.com` repository. Stable versions are found at the later one, and therefore I doubt whether we need it here.

Resource for further reading:
https://askubuntu.com/questions/185317/are-security-ubuntu-com-updates-eventually-merged-into-normal-updates/185444#185444

File: scripts/packages/chromium-package.sh (hunk 1 above)
Comment (abishekvashok) [reply scripts/packages/chromium-package.sh]: Okay @pranav1698

File: sources.xenial.list (hunk 2 above)
Comment (tabesin) [reply sources.xenial.list]: The repo should be ok. 

wget from the repo above is then obsolete.

Comment (pranav1698) [reply sources.xenial.list]: I have added this line if we want to use aptitude to install the package, the archive mirrors for the 32bit versions for the archive mirror are not in sync with the master repository, this is for the chromium repo as mentioned: https://packages.ubuntu.com/xenial/i386/chromium-browser/download
I have tested installing this package on my desktop with the archive mirror, as expected it took a long time to install, maybe to it was the syncing time I will still check this for travis
wget from the repo to get the package will solve all these problems as you can just get the package with the given url

**Comment by pranav1698** [comment]
The packages are installed without any problem and timing problems without the mirror changes 
https://github.com/pranav1698/meilix/releases/download/e/meilix-xenial-20190413-i386.iso
So, I think we should stick with suggestions by @abishekvashok

**Pull request merged**